README.md

```diff
@@ -1,24 +1,27 @@
-# README
+# Turning Green Core
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This repo contains the backend for various Turning Green stuff.
 
-Things you may want to cover:
+## Initial Setup
+For first-time users, do the following
+```
+1. Get Docker
+2. `docker-compose run web rails db:create`
+3. `docker-compose run web rails db:migrate`
+4. `docker-compose up`
```

Comment:
Need to add `docker-compose run web rake task:importjson` before starting up the server